index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="Navigation">
+    <div class = "nav-ban bg-dark">
+      <ul class="nb-list">
+        <li class="nb-item"><a class="a-nb-item" href="#"><img class= "fb" src="img/fb.svg"></a></li>
+        <li class="nb-item"><a class="a-nb-item" href="#"><img class= "twtr" src="img/twtr.svg"></a></li>
+        <li class="nb-item"><a class="a-nb-item" href="#">English</a></li>
+        <li class="nb-item"><a class="a-nb-item" href="#">My Page</a></li>
+        <li class="nb-item"><a class="a-nb-item" href="#">Logout</a></li>
+      </ul>
+    </div>
+    <div class="nav-int bg-white" >
+        
+      <div class="hamburger">
```

Comment:
- [ ] I think you can improve your mobile menu and make the mobile menu appears over the whole page, that will give a good user experience and make it more wonderful and I would like to recommend you be more consistent with the design (colors, typography, spacings, etc.) 👍 

![mobile menu](https://github.com/eamador1/business-summit/assets/6615394/7a3b0878-e419-4f2b-8cd1-c6e26fd79674)